pkg/provider/cloud/azure/client.go

```diff
@@ -158,6 +160,18 @@ func GetCredentialsForCluster(cloud kubermaticv1.CloudSpec, secretKeySelector pr
 	}, nil
 }
 
+func ValidateCredentials(credentials Credentials) error {
```

Comment:
Can we please have this as `ValidateCredentials(ctx context.Context credentials Credentials) error`?

Reply:
done